.github/workflows/CI.yml

```diff
@@ -26,12 +26,12 @@ jobs:
         arch:
           - x64
     steps:
-      - uses: actions/checkout@v3.5.0
-      - uses: julia-actions/setup-julia@v1
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2
         with:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
-      - uses: actions/cache@v2
+      - uses: actions/cache@v4
         env:
           cache-name: cache-artifacts
         with:
```

Comment:
There's a nice julia-specific cache action, so you can just replace this whole action block with `- uses: julia-actions/cache@v2`